careers/working-at-cfpb/index.html

```diff
@@ -0,0 +1,150 @@
+            {# TODO: Replace placeholder div with image. #}
+            <div class="hero_img"
+                 style="width:1200px;height:300px;background-image:url({{ 'http://placehold.it/1200x300.jpg ' }});">
+            </div>
+        </div>
+    </section>
+{%- endblock %}
+
+{% block content_main %}
+
+    <section class="block
+                    block__flush-top
+                    block__sub">
+            <h1>Working at the CFPB</h1>
+            <p class="h3">
+                Our mission stems from our commitment to serving the consumer
+                and our country.
+                Our vision is a consumer finance marketplace that works for
+                American consumers, responsible providers,
+                and the economy as a whole.
+            </p>
+            <p class="short-desc">
+                This vision can only come from the people who are working
+                toward it.
+                Creating this kind of market takes a dedication to service,
+                leadership and innovation.
+            </p>
+    </section>
+
+    {% import "macros/mobile-carousel.html" as mobile_carousel %}
+    {{ mobile_carousel.render({
+        'image_sizes_class': 'media_image__100',
+        'image_base_url':    '/static/img/',
+        'data': [{
+            'image_filename': 'serve.png',
+            'title':          'Serve',
+            'desc':      'Our mission begins with service to the
+                          consumer and our country.
+                          We serve our colleagues by listening to
+                          one another and by sharing our collective
+                          knowledge and experience.'
+            }, {
+            'image_filename': 'lead.png',
+            'title':          'Lead',
+            'desc':      'Fostering leadership and collaboration at
+                          all levels is at the core of our success.
+                          We believe in investing in the growth of
+                          our colleagues and in creating an
+                          organization that is accountable to the
+                          American people.'
+            }, {
+            'image_filename': 'innovate.png',
+            'title':          'Innovate',
+            'desc':      'Our organization embraces new ideas
+                          and technology.
+                          We are focused on continuously improving,
+                          learning, and pushing ourselves to be great.'
+            }]
+        })
+    }}
+
+    <div class="block
+                block__flush-top">
+        <p class="short-desc">As a bureau, we support these values by focusing on equal opportunity, diversity, and inclusion. We value candidates from all backgrounds, traditional and non-traditional. We believe in investing in the growth of our colleagues and in creating an organization that is accountable to and representative of the American people.</p>
```

Comment:
This needs to move into its own issue. The problem is this: We want paragraphs of body copy of substantial length—length of the whole section of copy, not length of an individual paragraph—to be set in Georgia. Pretty much all other paragraphs are fair game for Avenir. Back when we started CF, we thought that paragraphs of body copy were the "default" case, and as such, set that as the base font family, overriding with Avenir Next when necessary. This led to the case of the not-very-semantic class `.short-desc`, among other unfortunate things like `.styled-link` and `.micro-copy`. (See: [cf-typography docs](https://cfpb.github.io/cf-typography/docs/).)

Since then, we've definitely had second thoughts about reversing our position on what the base font family should be. What do you all think about changing the base font family to Avenir Next (i.e., the sans-serif stack), and creating a new rule that sets "long-read" body copy in Georgia? (serif stack).